ci/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM alpine:3.8 as conjur-test
```

Comment:
is there a reason not to use `alpine:latest` or at least `apline:3.11`?

Reply:
I moved the testing from Jenkins to github actions, so Dockerfile for test client pod was deleted.